lightningd/options.c

```diff
@@ -972,7 +974,7 @@ void handle_early_opts(struct lightningd *ld, int argc, char *argv[])
 				 ld->config_netdir);
 
 	/* Set default PID file name to be per-network */
-	ld->pidfile = tal_fmt(ld, "lightningd-%s.pid",
+	ld->pidfile = tal_fmt(ld, "../lightningd-%s.pid",
```

Comment:
No, because they get parsed next.

However, this "../" is wrong (network/ may be a symlink): this code is left over from before I gave in and made basedir and netdir explicit.  Will fix:

```C
	/* Set default PID file name to be per-network (in base dir) */
	ld->pidfile = path_join(ld, ld->config_basedir,
				tal_fmt(tmpctx, "lightningd-%s.pid",
					chainparams->network_name));
```